compiler/SynthesizeSymbolic.ml

```diff
@@ -8,8 +8,17 @@ open Errors
 
 let mk_mplace (span : Meta.span) (p : place) (ctx : Contexts.eval_ctx) : mplace
     =
-  let bv = Contexts.ctx_lookup_var_binder span ctx p.var_id in
-  { bv; projection = p.projection }
+  let rec place_to_projection_and_var (place : place) :
+      var_id * projection_elem list =
+    match place.kind with
+    | PlaceBase var_id -> (var_id, [])
+    | PlaceProjection (subplace, pe) ->
+        let var_id, proj = place_to_projection_and_var subplace in
+        (var_id, pe :: proj)
```

Comment:
We should compute the list of projection elements in the reverse order.
Also, I'm going to check if we really need to store this information.

Reply:
oh oops, you're right